components/TextInput.js

```diff
@@ -22,12 +22,13 @@ class TextInput extends PureComponent {
 
   handleBlur() {
     this.setState({ isFocused: false });
+    this.props.onBlur()
```

Comment:
Destructure `onBlur` from `this.props` on the first line of the method with a fallback to `null` if it isn't set. This should be followed by a newline. Then check if `onBlur` is truthy and run it:

```
handleBlur() {
  const { onBlur = null } = this.props;

  this.setState({ isFocused: false });
  if (onBlur) {
    onBlur();
  }
}
```